VRDR.Tests/DeathRecord_Should.cs

```diff
@@ -804,18 +804,18 @@ public void Get_INTERVAL1C()
             Assert.Equal("5 years", ((DeathRecord)XMLRecords[0]).INTERVAL1C);
         }
 
-        [Fact]
-        public void Set_CODE1C()
-        {
-            Dictionary<string, string> code = new Dictionary<string, string>();
-            code.Add("code", "code 3");
-            code.Add("system", "system 3");
-            code.Add("display", "display 3");
-            SetterDeathRecord.CODE1C = code;
-            Assert.Equal("code 3", SetterDeathRecord.CODE1C["code"]);
-            Assert.Equal("system 3", SetterDeathRecord.CODE1C["system"]);
-            Assert.Equal("display 3", SetterDeathRecord.CODE1C["display"]);
-        }
+        // [Fact]
+        // public void Set_CODE1C()
+        // {
+        //     Dictionary<string, string> code = new Dictionary<string, string>();
+        //     code.Add("code", "code 3");
+        //     code.Add("system", "system 3");
+        //     code.Add("display", "display 3");
+        //     SetterDeathRecord.CODE1C = code;
+        //     Assert.Equal("code 3", SetterDeathRecord.CODE1C["code"]);
+        //     Assert.Equal("system 3", SetterDeathRecord.CODE1C["system"]);
+        //     Assert.Equal("display 3", SetterDeathRecord.CODE1C["display"]);
+        // }
```

Comment:
delete?

Reply:
I'm leaving the CODE1X stuff pending Pete's review.